src/Pact/Analyze/AnalyzeNumerical.hs

```diff
@@ -0,0 +1,186 @@
+{-# LANGUAGE FlexibleContexts      #-}
+{-# LANGUAGE TypeFamilies          #-}
+{-# LANGUAGE GADTs                 #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE Rank2Types            #-}
+{-# LANGUAGE ScopedTypeVariables   #-}
+{-# LANGUAGE TypeOperators         #-}
+module Pact.Analyze.AnalyzeNumerical where
```

Comment:
Seems like this should probably be `Pact.Analyze.Analyze.Numerical`. And then once we rename our "analyze*" (vs translate, check, etc) to "eval*", then this would be `Pact.Analyze.Eval.Numerical` and core analyze/eval stuff would be in `Pact.Analyze.Eval`

Reply:
This use of the common convention would communicate that Pact.Analyze.Analyze is used by the outside world, and this new module is a some sub-implementation beneath that

Reply:
i agree. one todo for me is to figure out exactly what modules we should have. i think more than numerical should be broken out. like the rest of pure term analysis.